docs/audits/total-ad-blocking-time.md

```diff
@@ -0,0 +1,18 @@
+# Reduce Ad JS Blocking Time
+
+## Overview
+
+This audit measures how much [blocking time](https://web.dev/tbt/) is
+attributable to ads scripts. This captures time where the JS thread is blocked
+by an ad script for long enough to be noticeable by users when it impacts
```

Comment:
There should be a "More Information" section at the end that includes:

1. What constitutes "long enough" here
2. What ad scripts are supported

Should also probably call out when this measurement starts/stops.